api/datastore/sql/sql.go

```diff
@@ -276,14 +279,34 @@ func (ds *sqlStore) clear() error {
 	})
 }
 
+func (ds *sqlStore) GetAppID(ctx context.Context, appName string) (string, error) {
+	var app models.App
+	err := ds.Tx(func(tx *sqlx.Tx) error {
```

Comment:
we don't need the tx here for this query. this will add unnecessary overhead of starting and committing or rolling back a transation (3 step process instead of 1) and this transaction does not modify any state, we don't need read (or write) isolation since this query only does one read op.

Reply:
this one will be very 'hot'

Reply:
fixed